run_dir/static/js/project_samples.js

```diff
@@ -354,6 +354,13 @@ function load_all_udfs(){
         safeobj(key).html(auto_format(value));
       }
 
+      // highlight if project is shared
+      else if (prettify(key) == 'shared') {
+        if (value != 'No') {
+          $('#type').append('<span class="label label-warning">Shared</span>');
```

Comment:
But if you're using `append` then this is going after the previous label, right? So you'd need margin / a space on the left rather than the right? Either is fine - a space is probably easier :smile: